AWSAuthSDK/Sources/AWSUserPoolsSignIn/UserPoolsUI/AWSUserPoolsUIHelper.m

```diff
@@ -23,19 +23,44 @@ @implementation AWSUserPoolsUIHelper
 + (void) setUpFormShadowForView:(UIView *)view {
     view.layer.shadowColor = [UIColor blackColor].CGColor;
     view.layer.shadowOffset = CGSizeZero;
-    view.layer.shadowOpacity = 0.5;
-    view.layer.shadowRadius = 5;
+    view.layer.shadowOpacity = 0.25;
+    view.layer.shadowRadius = 6;
     view.layer.cornerRadius = 10.0;
-    view.layer.borderColor = [UIColor grayColor].CGColor;
+    view.layer.borderColor = [[UIColor grayColor] colorWithAlphaComponent:0.7].CGColor;
     view.layer.borderWidth = 0.5;
     view.layer.masksToBounds = NO;
 }
 
 + (UIColor *) getBackgroundColor:(id<AWSUIConfiguration>)config {
     if (config != nil && config.backgroundColor != nil) {
         return config.backgroundColor;
-    } else {
-        return [UIColor darkGrayColor];
+    } else if (@available(iOS 13.0, *)) {
+        return [UIColor systemBackgroundColor];
+    }
+    return [UIColor darkGrayColor];
+}
+
++ (UIColor *) getDefaultBackgroundColor {
+    if (@available(iOS 13.0, *)) {
+        return [UIColor secondarySystemBackgroundColor];
```

Comment:
Why is the background color we choose from the system here in `getDefaultBackgroundColor` different from the background color we choose from the system in `getBackgroundColor`?